### PR TITLE
Update Assistant-related icons

### PR DESCRIPTION
**Pull request by derekblank**
Improves Assistant-related icons to match designs. 

## Proposed Changes

- Removes static `<ReturnIcon />` and `<MenuIcon />` in favor of `@wordpress/icons`: `keyboardReturn` and `moreVertical`.
- Updates `<AssistantIcon />` to regular function component ([#](https://github.com/Automattic/studio/pull/202#discussion_r1626074876)) 

## Testing Instructions

1. Start app with AI Assistant flag enabled: `STUDIO_AI=true npm start`
2. Navigate to Assistant tab
3. **Type text** in the `<AIInput />` component
4. Observe the following icons (left to right -- `Assistant`, `keyboardReturn`, and `moreVertical`):

Icons

## Pre-merge Checklist

- [x] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: src/components/ai-input.tsx [hunk 1]
```diff
@@ -98,18 +97,18 @@ export const AIInput = ( {
 					data-testid="ai-input-textarea"
 				/>
 				{ input.trim() !== '' && (
-					<div className="flex items-end py-4 mb-1">
-						<ReturnIcon />
+					<div className="flex items-end py-4 mb-1 ">
```
Comment (dcalhoun) [inline src/components/ai-input.tsx]:
```suggestion
					<div className="flex items-end py-4 mb-1">
```

File: src/components/ai-input.tsx [hunk 2]
```diff
@@ -98,18 +97,18 @@ export const AIInput = ( {
...
+						<Icon icon={ keyboardReturn } size={ 13 } fill="#cccccc" />
 					</div>
 				) }
-				<div className="flex items-end p-2">
+				<div className="flex items-end py-2">
```
Comment (dcalhoun) [inline src/components/ai-input.tsx]: The horizontal padding removal creates collision between the input and button focus styles, I feel space between the elements is beneficial. I wonder if we might retain the padding or address this in a different manner. WDYT? 

Semi-related: I note the focus style color is different between these two elements. 

Overlapping input and button focus styles

**Review by dcalhoun — APPROVED** [review]
The new icons look good to me. 🚀

File: src/components/ai-input.tsx (hunk 2 above)
Comment (derekblank) [reply src/components/ai-input.tsx]: Good catch. I transferred the button spacing to the margin to preserve the unfocused layout styles, and updated the AIInput focus color to match the Button focus color (and the rest of Studio's input components). 👍 

https://github.com/Automattic/studio/assets/643285/2dd1890c-b078-404a-bbe9-abf7d4beb56d

**Pull request merged**